app/src/main/java/com/cornellappdev/transit/ui/viewmodels/RouteViewModel.kt

```diff
@@ -0,0 +1,26 @@
+
+package com.cornellappdev.transit.ui.viewmodels
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import com.cornellappdev.transit.models.Route
+import com.cornellappdev.transit.models.RouteRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import javax.inject.Inject
+
+@HiltViewModel
+class RouteViewModel @Inject constructor(
+    private val repository: RouteRepository,
+    private val savedState: SavedStateHandle
+) : ViewModel() {
+    //TODO when savedStateHandle is set up, get info from it which is populated thru nav
+
+    val startPl = "Current Location"
+
+    val destPl = "Destination"
+
+    val time = "12:00AM"
```

Comment:
We should make these into Flows instead down the line. But we'll probably do that after we settle on a data representation for stops, locations, current location, and their relationships.

...At least mark that with a TODO!